bucoffea/plot/util.py

```diff
@@ -214,6 +214,7 @@ def merge_datasets(histogram):
     # Some combinations are the same for all years
     yearly = {
         'GJets_HT_MLM_{year}' : 'GJets_HT-(\d+)To.*-MLM_{year}',
+        'GJets_DR-0p4_HT_MLM_{year}' : 'GJets_DR-0p4_HT-(\d+)To.*-MLM_{year}',
```

Comment:
I don't think this regular expression captures all the DR datasets:

```
GJets_DR-0p4_HT-100To200-MLM_2016
GJets_DR-0p4_HT-100To200-MLM_2018
GJets_DR-0p4_HT-100To200-MLM_v2_2017
GJets_DR-0p4_HT-200To400-MLM_2016
GJets_DR-0p4_HT-200To400-MLM_2018
GJets_DR-0p4_HT-200To400-MLM_v2_2017
GJets_DR-0p4_HT-400To600-MLM_2016
GJets_DR-0p4_HT-400To600-MLM_2018
GJets_DR-0p4_HT-400To600-MLM_v2_2017
GJets_DR-0p4_HT-40To100-MLM_2016
GJets_DR-0p4_HT-600ToInf-MLM_2016
GJets_DR-0p4_HT-600ToInf-MLM_2017
GJets_DR-0p4_HT-600ToInf-MLM_2018
GJets_DR-0p4_HT-600ToInf-MLM_2018
```

Reply:
You're right, I didn't look into 2017/18 samples. I will fix it now, thanks for catching it.